<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonsbOperateWin</class>
 <widget class="QDialog" name="MonsbOperateWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="4,5,2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Monsb Controls - Cancels Table</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QComboBox" name="MonsbTypeOperatesCb">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="MonsbOperatesTv">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,1">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Controls</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QTableWidget" name="ControlsTv">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="ControlsGb">
         <property name="title">
          <string>menuControls</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QPushButton" name="ControlsDefaultBn">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>default</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ControlsAddMultiBn">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>append</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ControlsInsertMultiBn">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>insert</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ControlsAddPairBn">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>&gt; pair &lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,1">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Cancels</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QTableWidget" name="CancelsTv">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="CancelsGb">
         <property name="title">
          <string>menuCancels</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QPushButton" name="CancelsDefaultBn">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>default</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="CancelsAddMultiBn">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>append</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="CancelsInsertMultiBn">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>insert</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="CancelsAddPairBn">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>&gt; pair &lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="CurrentMonsbOperateLb">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>......</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Monsbs</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QComboBox" name="MonsbTypeSelectsCb">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="MonsbSelectsTv">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
